src/components/Hero.tsx

```diff
@@ -104,9 +105,13 @@ const Hero = () => {
 
                     <div className="flex justify-between mt-[8px]">
                       <div className="flex gap-x-[5px]">
-                        <div className="w-[20px] h-[20px] rounded-full">
-                          {/* todos: add contributors avator here */}
-                        </div>
+                        {contributions.slice(0, 3).map(contribution => (
+                          <div className="w-[20px] h-[20px] rounded-full">
```

Comment:
Also add a key while mapping through an array of items, This way React can handle the minimal DOM change.

Reply:
sounds like this is fixed on the beta branch with the new linter. Thanks for catching @SunGoldTech